.github/actions/setup-dependencies/action.yaml

```diff
@@ -12,7 +12,7 @@ runs:
       - name: Setup Go
         uses: actions/setup-go@v5
         with:
-          go-version: '1.22'
+          go-version: '1.23.6'
```

Comment:
question: should we be using the redhat fips golang directly? Is it enough to use the same version upstream? I guess why should upstream  not continue to use latest version of 1.23?

Reply:
if this last attempt will not work will move to 1.23

Reply:
Current release is 1.24.0, so 1.22 has no official support (excluding critical security problems).

[Release Policy](https://go.dev/doc/devel/release#policy)
Each major Go release is supported until there are two newer major releases. For example, Go 1.5 was supported until the Go 1.7 release, and Go 1.6 was supported until the Go 1.8 release. We fix critical problems, including [critical security problems](https://go.dev/security), in supported releases as needed by issuing minor revisions (for example, Go 1.6.1, Go 1.6.2, and so on).